### PR TITLE
Issue #SB-20637 feat:fixed controller logging

### PR DESCRIPTION
**Pull request by amit-tarento**
Fixed controller logging

### CODE REVIEW

File: ansible/roles/stack-sunbird/templates/learner-service_logback.xml [hunk 1]
```diff
@@ -6,8 +6,25 @@
...
+          <timestamp>timestamp</timestamp>
+          <message>msg</message>
+          <logger>lname</logger>
+          <thread>tname</thread>
+          <levelValue>[ignore]</levelValue>
+          <version>[ignore]</version>
+          <stack_trace>exception</stack_trace>
+        </fieldNames>
+        <throwableConverter class="net.logstash.logback.stacktrace.ShortenedThrowableConverter">
+          <maxDepthPerThrowable>30</maxDepthPerThrowable>
```
Comment (SMYALTAMASH) [inline ansible/roles/stack-sunbird/templates/learner-service_logback.xml:21]: can we variablize these values @amit-tarento ?

File: ansible/roles/stack-sunbird/templates/learner-service_logback.xml [hunk 2]
```diff
@@ -6,8 +6,25 @@
...
+          <maxLength>2048</maxLength>
```
Comment (SMYALTAMASH) [inline ansible/roles/stack-sunbird/templates/learner-service_logback.xml:22]: can we variablize these values @amit-tarento ?

File: ansible/roles/stack-sunbird/templates/learner-service_logback.xml (hunk 1 above)
Comment (amit-tarento) [reply ansible/roles/stack-sunbird/templates/learner-service_logback.xml:21]: We can , But not required as its just to print exception stacktrace , with limited verbose.

File: ansible/roles/stack-sunbird/templates/learner-service_logback.xml (hunk 2 above)
Comment (amit-tarento) [reply ansible/roles/stack-sunbird/templates/learner-service_logback.xml:22]: We can , But not required as its just to print exception stacktrace , with limited verbose.

**Review by SMYALTAMASH — APPROVED** [review]

**Pull request merged**